tools/check-git-version.js

```diff
@@ -10,7 +10,9 @@ function checkGitVersion() {
       .stdout.toString();
     gitVersion = gitVersion.match(regex)[0];
     if (semver.lt(gitVersion, GIT_MINIMUM_VERSION)) {
-      throw new Error('Minimum git version 2.33.0 is required');
+      throw new Error(
+        'Minimum git version ' + GIT_MINIMUM_VERSION + ' is required'
```

Comment:
```suggestion
        'Minimum Git version ' + GIT_MINIMUM_VERSION + ' is required'
```

Git is a proper noun, should be capitalized. 😉

Reply:
```suggestion
        `Minimum git version ${GIT_MINIMUM_VERSION} is required`
```

Use javascript string templates